lsp/server/src/server.ts

```diff
@@ -31,9 +37,13 @@ let hasWorkspaceFolderCapability = false;
 export let hasDiagnosticRelatedInformationCapability = false;
 
 let extensionName: string = '';
+let workspaceFolder: string | null | undefined;
```

Comment:
I don't see `workspaceFolder` being used anywhere except for it getting a value assigned in line 46.

In case I missed it's usage somewhere in this file I just want to know does `workspaceFolder` need to be `string | null | undefined`? Or can it simply be just `string` or maybe even `string | null` or `string | undefined`?